### ISSUE TITLE
MRH Satellite glitches to the side of the mountain in view

### ISSUE THREAD

**Issue by Will-Nichols** [Task-Force-Red/Operation-Python#45]
Satellite was working and then it decided to glitch into a view where it looked like the satellite was literally landed on the side of the mountain. No one was able to reposition the satellite.

**Comment by jerkinmerkin** [comment]
Did you try adjusting the zoom while in full screen mode?  Use the slider on the right.

**Comment by Will-Nichols** [comment]
Yes and it would not function. What is the height set at for the satellite?
